<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paramsCv</class>
 <widget class="QWidget" name="paramsCv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameter Vission</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color:  rgb(85, 170, 127);
    min-width: 10em;
    padding: 6px;
}
</string>
   </property>
   <property name="title">
    <string>threshold</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>k-thresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_kThresh">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>blockSize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_blockSize">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_C">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_useOtsu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>useOtsu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_normal">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>normal</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color:  rgb(85, 170, 127);

    min-width: 10em;
    padding: 6px;
}</string>
   </property>
   <property name="title">
    <string>morphology</string>
   </property>
   <widget class="QLineEdit" name="ln_iter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_kSize">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>k-size</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color:  rgb(85, 170, 127);

    min-width: 10em;
    padding: 6px;
}</string>
   </property>
   <property name="title">
    <string>removeBlobs</string>
   </property>
   <widget class="QLineEdit" name="ln_height">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>area</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_width">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_area">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color:  rgb(85, 170, 127);

    min-width: 10em;
    padding: 6px;
}</string>
   </property>
   <property name="title">
    <string>OCR</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>oem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>psm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_oem">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0    Legacy engine only.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1    Neural nets LSTM engine only.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2    Legacy + LSTM engines.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3    Default, based on what is available.</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbb_psm">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0    Orientation and script detection (OSD) only.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1    Automatic page segmentation with OSD.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2    Automatic page segmentation, but no OSD, or OCR.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3    Fully automatic page segmentation, but no OSD. (Default)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4    Assume a single column of text of variable sizes.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5    Assume a single uniform block of vertically aligned text.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6    Assume a single uniform block of text.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7    Treat the image as a single text line.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8    Treat the image as a single word.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9    Treat the image as a single word in a circle.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10    Treat the image as a single character.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11    Sparse text. Find as much text as possible in no particular order.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12    Sparse text with OSD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13    Raw line. Treat the image as a single text line,bypassing hacks that are Tesseract-specific.</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lang</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_lang">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>eng</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>chi_sim</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jpn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>vie</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
